### ISSUE TITLE
'dotnet11' verbs need to set 'fusion' as native override after install ("Failed to delay load library mscorlib.dll")

### ISSUE THREAD

**Issue by rmi1974** [Winetricks/winetricks#1676]
Hello folks,

not sure when this was tested the last time ... 
It had to be a long time ago someone tried to run .NET 1.x stuff.

See also bug:

https://bugs.winehq.org/show_bug.cgi?id=25120 ("Microsoft .NET Framework 1.x installer fails with 'CLR error: 80004001' (1.0) or 'Failed to delay load library mscorlib.dll' (1.1) (fusion.CreateApplicationContext is a stub)")

which refers to installer.

But any .NET 1.x app will fail to run as well afterwards:

https://source.winehq.org/git/wine.git/commitdiff/d366642519e908556a9c425285d21e78f0a1e101 ("fusion: No longer prefer native.") -> part of Wine 1.5.22 release

```
$ WINEDEBUG=+loaddll,+msgbox wine ./DSLWatch.exe 
...
trace:loaddll:load_native_dll Loaded L"C:\\windows\\Microsoft.NET\\Framework\\v1.1.4322\\MSVCR71.dll" at 0x7c340000: native
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\psapi.dll" at 0x7df00000: builtin
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\dbghelp.dll" at 0x7df10000: builtin
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\fusion.dll" at 0x7df70000: builtin
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\comctl32.dll" at 0x7dbe0000: builtin
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\shell32.dll" at 0x7dce0000: builtin
trace:loaddll:load_native_dll Loaded L"C:\\windows\\Microsoft.NET\\Framework\\v1.1.4322\\mscorwks.dll" at 0x791b0000: native
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\uxtheme.dll" at 0x7dbb0000: builtin
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\rpcrt4.dll" at 0x7d9e0000: builtin
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\ole32.dll" at 0x7da60000: builtin
fixme:sync:CreateMemoryResourceNotification (0) stub
fixme:fusion:CreateApplicationContext 0x17b3c8, 0x793e71a8
trace:loaddll:load_builtin_dll Loaded L"C:\\windows\\system32\\oleaut32.dll" at 0x7d8b0000: builtin
trace:msgbox:MSGBOX_OnInit L"Failed to delay load library mscorlib.dll (Win32 error: 0).\n\nThis program can no longer run and will now terminate."
```

Example app: DSLWatch .NET 1.6

https://web.archive.org/web/20051023162214/http://www.xwatch.de/download/dslwnet.exe

Source:

https://github.com/Winetricks/winetricks/blob/3666dab60b494c614278331386f74c84864ef3e5/src/winetricks#L8609

https://github.com/Winetricks/winetricks/blob/3666dab60b494c614278331386f74c84864ef3e5/src/winetricks#L8669

(SP1 probably as well).

I didn't look if **verify_dotnet11** actually tries to run a .NET 1.x app
If yes then the check is not sufficient enough for real .NET 1.x apps.

Regards

**Comment by austin987** [comment]
`verify_dotnet11` calls `w_dotnet_verify dotnet11`, which runs dotnet_verifier.

I started a PR a while back to try to get some real binaries with mcs, but could never figure out the right flags, see #1459. Maybe you've got some ideas (or some way to do it with dotnetsdk or something similar). Either way (using dotnet/mono to compile), I'd like to get some basic hello world dotnet tests.

**Comment by austin987** [comment]
Fixed by 7002017, thanks for the report.

If you have any ideas/feedback on the dotnet binary idea, I'd love any feedback you might have.
